lib/matplotlib/table.py

```diff
@@ -14,7 +14,7 @@
 You can add additional cells outside this range to have convenient
 ways of positioning more interesting grids.
 
-Author    : John Gill <jng@europe.renre.com>
```

Comment:
Can you explain why not to change the email?

You realise I am the original author?

Is there some reason why you want the code to have an ancient email address for me that no longer works?